docs/examples/polytope.ipynb

```diff
@@ -54,146 +59,800 @@
     },
     "tags": []
    },
+   "outputs": [],
+   "source": [
+    "import earthkit.data\n",
+    "\n",
+    "LOCATION = ((-9.11, 38.79))\n",
+    "\n",
+    "request = {\n",
+    "    \"class\": \"od\",\n",
+    "    \"stream\" : \"enfo\",\n",
+    "    \"type\" : \"pf\",\n",
+    "    \"date\" : -1,\n",
+    "    \"time\" : \"0000\",\n",
+    "    \"levtype\" : \"sfc\",\n",
+    "    \"expver\" : \"0001\", \n",
+    "    \"domain\" : \"g\",\n",
+    "    \"param\" : \"164/167/169\",\n",
+    "    \"number\" : \"1/to/50\",\n",
+    "    \"step\": \"0/to/360\",\n",
+    "    \"feature\" : {\n",
+    "        \"type\" : \"timeseries\",\n",
+    "        \"points\": [[LOCATION[0], LOCATION[1]]],\n",
+    "        \"axes\": \"step\",\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "ds = earthkit.data.from_source(\n",
+    "    \"polytope\",\n",
+    "    \"ecmwf-mars\",\n",
+    "    request,\n",
+    "    stream=False,\n",
+    "    address='polytope.ecmwf.int',\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c232c51b",
+   "metadata": {},
+   "source": [
+    "The collection being accessed is `ecmwf-mars`. The endpoint being accessed is `polytope.ecmwf.int`.\n",
+    "\n",
+    "To view the downloaded coveragejson you can run teh following cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "32cb0e10-7545-4758-b2f0-99984f01d71f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds._json()"
```

Comment:
_json should not be advertised. If we need to expose it a public method should added in a separate PR.

Reply:
does it make sense to expose it? I would think so, but not sure if this needs to be consistent with other data sources?

Reply:
Not sure what the method name should be and what it should return. A dict?